.github/workflows/staging_e2e.yaml

```diff
@@ -36,12 +36,12 @@ jobs:
           echo "COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
           GOLANG_VERSION=$(./hack/golang-version.sh)
           echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION := }" >> $GITHUB_ENV
-          echo "HOLODECK_VCENTER_USERNAME=${HOLODECK_VCENTER_USERNAME}" >> $GITHUB_ENV
-          echo "HOLODECK_VCENTER_PASSWORD=${HOLODECK_VCENTER_PASSWORD}" >> $GITHUB_ENV
-          cat /home/runner/.ssh/vsphere-key > ${GITHUB_WORKSPACE}/holodeck_ssh_key
       - name: Set up Holodeck
-        uses: NVIDIA/holodeck@v0.2.2
+        uses: NVIDIA/holodeck@v0.2.3
         with:
+          vsphere_username: ${{ secrets.VSPHERE_USERNAME }}
```

Comment:
Just so it's clear: This means that v0.2.3 of holodeck includes `INPUT_` variables for all these values?